xml/net_bonding.xml

```diff
@@ -229,12 +230,13 @@
           STARTMODE='auto' # or 'onboot'
           BOOTPROTO='static'
           IPADDR='192.168.0.1/24'
+          <!-- 2022-04-01 tbazant: update config options as well? how? -->
```

Comment:
as far as i know wicked has not introduced inclusive naming compatible replacement yet.